packages/arb-token-bridge-ui/src/pages/api/withdrawals.ts

```diff
@@ -9,6 +9,7 @@ type NextApiRequestWithWithdrawalParams = NextApiRequest & {
     l2ChainId: string
     address: string
     search?: string
+    fetchSentTx?: string
```

Comment:
The API route seems like the wrong place for this abstraction. Let's add support for a few more query params:

```suggestion
    sender?: string
    senderNot?: string
    receiver?: string
    receiverNot?: string
```

I've added `receiverNot` for completeness. Let's keep keep them camel case to match the rest.

---

After we have this, we can add some code in the UI that builds the right query params for us, e.g:

```
function getQueryParamsForFetchingSentFunds(address: string)
function getQueryParamsForFetchingReceivedFunds(address: string)

// then use it like this

await fetchDeposits({
  walletAddress: '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045',
  ...getQueryParamsForFetchingReceivedFunds(walletAddress)
})
```

It's a bit more verbose but I think it'll make it easier to reason about the code in the long run.